src/Kiota.Builder/Writers/Dart/CodeMethodWriter.cs

```diff
@@ -206,7 +200,7 @@ private void WriteFactoryMethodBodyForIntersectionModel(CodeMethod codeElement,
         {
             if (property.Type is CodeType propertyType)
             {
-                var typeName = conventions.GetTypeString(propertyType, codeElement, true, false);
+                conventions.GetTypeString(propertyType, codeElement, true, false);
```

Comment:
why are we not assigning the result of that call? should the call be removed entirely?

Reply:
Yeah you're right, adjusted